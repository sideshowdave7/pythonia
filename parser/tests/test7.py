"bob"[1]
x = {1:2, 3:4, }
def foo(a, b,):
 pass
def bar(a,):
 return 3
def baz():
    return {a, b,}
y= 4

