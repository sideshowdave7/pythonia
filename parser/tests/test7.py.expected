(program
 (expr (indexed "bob" (subscript 1)))
 (= (x) (dict (1 2) (3 4)))
 (def (foo a b) (suite (pass)))
 (def (bar a) (suite (return 3)))
 (def (baz) (suite (return (set a b))))
 (= (y) 4))

